scale/queue/test/test_models.py

```diff
@@ -507,187 +379,3 @@ def test_successful_ingest(self, mock_msg_mgr):
-        recipe_b = recipe_test_utils.create_recipe(recipe_type=recipe_type_b, input=data_b)
-        self.job_b_1 = job_test_utils.create_job(job_type=job_type_b_1, status='FAILED', input=data_dict,
-                                                 recipe=recipe_b)
-        self.job_b_2 = job_test_utils.create_job(job_type=job_type_b_2, status='CANCELED', num_exes=0, recipe=recipe_b)
-        self.job_b_3 = job_test_utils.create_job(job_type=job_type_b_3, status='BLOCKED', num_exes=0, recipe=recipe_b)
-
-        recipe_test_utils.create_recipe_job(recipe=recipe_b, job_name='Job 1', job=self.job_b_1)
-        recipe_test_utils.create_recipe_job(recipe=recipe_b, job_name='Job 2', job=self.job_b_2)
-        recipe_test_utils.create_recipe_job(recipe=recipe_b, job_name='Job 3', job=self.job_b_3)
-
-        # Job IDs to re-queue
-        self.job_ids = [self.standalone_failed_job.id, self.standalone_canceled_job.id,
-                        self.standalone_completed_job.id, self.job_a_1.id, self.job_b_2.id,
-                        self.standalone_queued_job.id]
-
-    def test_successful(self):
-        """Tests calling QueueManager.requeue_jobs() successfully"""
-
-        status = Queue.objects.get_queue_status()
-        self.assertEqual(status[0].count, 1)
-
-        Queue.objects.requeue_jobs(self.job_ids, self.new_priority)
-
-        standalone_failed_job = Job.objects.get(id=self.standalone_failed_job.id)
-        self.assertEqual(standalone_failed_job.status, 'QUEUED')
-        self.assertEqual(standalone_failed_job.max_tries, 6)
-
-        standalone_canceled_job = Job.objects.get(id=self.standalone_canceled_job.id)
-        self.assertEqual(standalone_canceled_job.status, 'QUEUED')
-        self.assertEqual(standalone_canceled_job.max_tries, 4)
-
-        # Superseded job should not be re-queued
-        standalone_superseded_job = Job.objects.get(id=self.standalone_superseded_job.id)
-        self.assertEqual(standalone_superseded_job.status, 'FAILED')
-
-        # Completed job should not be re-queued
-        standalone_completed_job = Job.objects.get(id=self.standalone_completed_job.id)
-        self.assertEqual(standalone_completed_job.status, 'COMPLETED')
-
-        job_a_1 = Job.objects.get(id=self.job_a_1.id)
-        self.assertEqual(job_a_1.status, 'QUEUED')
-        job_a_2 = Job.objects.get(id=self.job_a_2.id)
-        self.assertEqual(job_a_2.status, 'BLOCKED')
-
-        job_b_1 = Job.objects.get(id=self.job_b_1.id)
-        self.assertEqual(job_b_1.status, 'FAILED')
-        job_b_2 = Job.objects.get(id=self.job_b_2.id)
-        self.assertEqual(job_b_2.status, 'BLOCKED')
-        job_b_3 = Job.objects.get(id=self.job_b_3.id)
-        self.assertEqual(job_b_3.status, 'BLOCKED')
-
-        # check queue status
-        status = Queue.objects.get_queue_status()
-        sum = 0
-        for s in status:
-            sum += s.count
-        self.assertEqual(sum, 5)
-
-        canceled = Queue.objects.filter(is_canceled=True)
```

Comment:
This is good, but we need to move the test I added for requeueing a queued job to the queue_job test.